modules/ols-custom-resource-definition-file-parameters.adoc

```diff
@@ -0,0 +1,498 @@
+// Module included in the following assemblies:
+
+// * about/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ols-custom-resource-definition-file-parameters_{context}"]
+= Lightspeed Custom Resource Definition specification
+
+The following tables provide information about {ols-offical} Custom Resource Definition (CRD) specification 
```

Comment:
The following tables provide information about OLSConfig Custom Resource Definition (CRD) specification

Reply:
done